examples/meteringlab/c1/cluster.yaml

```diff
@@ -0,0 +1,183 @@
+# Namespace where the Scylla Cluster will be created
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: scylla
+
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: xfs-nvme-249g-iotune
+  namespace: scylla
+data:
+  io_properties.yaml: |
+    disks:
+    - mountpoint: /var/lib/scylla
+      read_iops: 385400
+      read_bandwidth: 3087631104
+      write_iops: 240628
+      write_bandwidth: 2289281280
+
+---
+
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: ScyllaDBMonitoring
+metadata:
+  name: example
+spec:
+  type: Platform
+  endpointsSelector:
+    matchLabels:
+      app.kubernetes.io/name: scylla
+      scylla-operator.scylladb.com/scylla-service-type: identity
+      scylla/cluster: scylla-cluster
+  components:
+    prometheus:
+      storage:
+        volumeClaimTemplate:
+          spec:
+            resources:
+              requests:
+                storage: 1Gi
+    grafana:
+      exposeOptions:
+        webInterface:
+          ingress:
+            ingressClassName: haproxy
+            dnsDomains:
+            - grafana.meteringlab.svc.cluster.local
```

Comment:
you will use this domain in second k8s cluster, usually `*.cluster.local` is internal k8s dns domain, you need something that is external to kube.